### ISSUE TITLE
Typing error when using enableInExpoDevelopment in Sentry.Init with TypeScript

### ISSUE THREAD

**Issue by brooksbecton** [expo/sentry-expo#76]
I'm seeing a typing issue when using TypeScript with this project. 
I see that y'all are using sentry-react-native types here: 
https://github.com/expo/sentry-expo/blob/master/index.d.ts#L1

But there are some expo specific configurations that are needed to use Sentry in Dev with Expo. 
Specifically, `enableInExpoDevelopment `
https://docs.expo.io/versions/latest/guides/using-sentry/#install-and-configure-sentry

It still works if you ignore the typings with `//@ ts-ignore` but the typing is wrong. 

How about this? 
```js
import * as reactNativeSentry from "@sentry/react-native";

type expoSentry extends reactNativeSentry {
    enableInExpoDevelopment: boolean
}
```

**Comment by nehresma** [comment]
PR #75 addresses this.  Just hoping it will get someone's attention and get merged.

**Comment by cruzach** [comment]
Merged and published in v2.0.1 😄
